### PR TITLE
Dev merge request

### PR DESCRIPTION
**Pull request by maelfosso**
Hi,

Please could you review our code ?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+          </div>
+        </div>
+
+        <div class="footer-2">
+          <!-- <div class="container"> -->
+            <div class="cols jc-sb ai-c">
+              <div>
+                <a href="#">
+                  <i class="icon heart"></i>
+                  <svg class="logo-svg" viewBox="0 0 103 29">
```
Comment (JamesOkunlade) [inline index.html:187]: This svg path is not needed. You have it on line 15 already.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+    }
+
+    .center-text{
+        margin: auto;
+    }
+
+}
+
+@media screen and (max-width:800px) {
+    .header{
```
Comment (JamesOkunlade) [inline css/main.css]: Remove selector and commented style if not needed.

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+}
+
+.footer {
+  background: #2b313e;
+  color: white;
+}
+
+.footer .col-item input {
+  height: 100%;
+  /* padding: .5rem; */
```
Comment (JamesOkunlade) [inline css/main.css]: Remove commented style if not needed. Don't leave unused style as comments in your file.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
Good job working on your last HTML & CSS project. There are commits from just one person, there should be evenly distributed commits from both of you on every project. 

Nice job on the Pixenio clone, looks almost perfect in your first try. :sunglasses: Just few things you have to work on.

1. Relative to the main pixenio site, the main body section of your content should be centralized with wide margins on both sides. In your 12 column grid framework, the body content should take the center 8 columns with 2 columns margin each on left and right sides of the body container.

2. Also relative to the main site, there should be some good margins between your header and main body as well. 

3. The div with class name "section center-text" content should be aligned to center.

4. Ensure the correct visual hierachy of the contents in the div with class name "smake-content" , i.e. the body weight of h2 and p are reflect what's on the main site.

5. I don't see the social media icons in your footer. Try to work on this and ensure margin between the left and right footer as well.

6. Refactor your code according to my  different comments on different lines of code.

When you're done with these changes, request for another code review and add me(James) and the TSE so I can follow up.

Happy coding!

File: index.html (hunk 1 above)
Comment (maelfosso) [reply index.html:187]: As it's present both on header and footer, is it possible to replicate it without duplicate the code ?

**Review by JamesOkunlade — APPROVED** [review]
### Awesome work guys!!! :sunglasses:

Merge the dev to master, close the PR and fill the completion form.

Here are your thumbs up. :+1::+1: 

Good luck in your Ruby section!

**Comment by maelfosso** [comment]
Thank you

**Pull request merged**